runtime/autoload/provider/python3.vim

```diff
@@ -1,45 +1,16 @@
-" The Python3 provider uses a Python3 host to emulate an environment for running
-" python3 plugins. :help provider
-"
-" Associating the plugin with the Python3 host is the first step because
-" plugins will be passed as command-line arguments
-
 if exists('g:loaded_python3_provider')
   finish
 endif
-let [s:prog, s:err] = provider#pythonx#Detect(3)
-let g:loaded_python3_provider = empty(s:prog) ? 1 : 2
+let g:loaded_python3_provider = 1
```

Comment:
It gets overridden below (L15) but then why would we need to declare this here?

Reply:
Idk, didn't wanna forget it in case the one on line 15 was removed. I'll remove it.